tests/test_meshing_queries_fdl.py

```diff
@@ -2025,7 +2025,7 @@ def test_meshing_utilities(new_mesh_session):
 
     assert meshing_session.meshing_utilities.replace_cell_zone_suffix(
         cell_zone_id_list=[87], old_suffix="fluid", new_suffix="fluid-new", merge=True
-    ) == [False]
+    ) == ["*the-non-printing-object*"]
```

Comment:
Should this be fixed on Fluent side?

Reply:
@mkundu1 It was returning [False] but unexpectedly now returns ["*the-non-printing-object*"] 

![image](https://github.com/ansys/pyfluent/assets/106588300/54281e7d-6957-4081-9a01-90e465b88d13)

As per last discussion in PyFluent Coordination meeting it should get fixed earlier but no idea why it is still there.